api/ws/server.go

```diff
@@ -219,7 +221,9 @@ func (w *WebSocketServer) AcceptBlock(b *chain.StatefulBlock) error {
 		if err != nil {
 			return err
 		}
-		w.s.Publish(append([]byte{TxMode}, bytes...), listeners)
```

Comment:
Really not a fan of not using the blank return for errors in the old code... feels like it makes it non-obvious that we're ignoring error values.